docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -155,24 +164,24 @@ processed after a dynamic filter is pushed down to the table scan.
 Dynamic filter collection thresholds
 ------------------------------------
 
-In order for dynamic filtering to work, the smaller dimension table
-needs to be chosen as a join’s build side. The cost-based optimizer can automatically
-do this using table statistics provided by connectors. Therefore, it is recommended
-to keep :doc:`table statistics </optimizer/statistics>` up to date and rely on the
-CBO to correctly choose the smaller table on the build side of join.
-
-Collection of values of the join key columns from the build side for
-dynamic filtering may incur additional CPU overhead during query execution.
-Therefore, to limit the overhead of collecting dynamic filters
-to the cases where the join operator is likely to be selective,
-Trino defines thresholds on the size of dynamic filters collected from build side tasks.
-Collection of dynamic filters for joins with large build sides can be enabled
-using the ``enable-large-dynamic-filters`` configuration property or the
+In order for dynamic filtering to work, the smaller dimension table needs to be
+chosen as a join’s build side. The cost-based optimizer can automatically do
+this using table statistics provided by connectors. Therefore, it is recommended
+to keep :doc:`table statistics </optimizer/statistics>` up to date and rely on
+the CBO to correctly choose the smaller table on the build side of join.
+
+Collection of values of the join key columns from the build side for dynamic
+filtering may incur additional CPU overhead during query execution. Therefore,
+to limit the overhead of collecting dynamic filters to the cases where the join
+operator is likely to be selective, Trino defines thresholds on the size of
+dynamic filters collected from build side tasks. Collection of dynamic filters
+for joins with large build sides can be enabled using the
+``enable-large-dynamic-filters`` configuration property or the
 ``enable_large_dynamic_filters`` session property.
 
-When large dynamic filters are enabled, limits on the size of dynamic filters can
-be configured for each join distribution type using the configuration properties
-``dynamic-filtering.large-broadcast.max-distinct-values-per-driver``,
+When large dynamic filters are enabled, limits on the size of dynamic filters
+can be configured for each join distribution type using the configuration
+properties ``dynamic-filtering.large-broadcast.max-distinct-values-per-driver``,
 ``dynamic-filtering.large-broadcast.max-size-per-driver`` and
 ``dynamic-filtering.large-broadcast.range-row-limit-per-driver`` and their
 equivalents for partitioned join distribution type.
```

Comment:
Is it worth putting these long, very similar property names in a bulleted list, to help distinguish them?